ci-operator/config/openshift-kni/commatrix/openshift-kni-commatrix-main.yaml

```diff
@@ -86,7 +86,7 @@ tests:
         oc krew install --manifest=cmd/commatrix-krew.yaml
 
         # run the make e2e-test
-        go mod vendor && make e2e-test
+        go mod vendor && PLATFORM_TYPE=aws make e2e-test
```

Comment:
What's the reason for the hardcoded value `PLATFORM_TYPE=aws` here?

Reply:
to use it on the code